lib/ansible/modules/notification/rabbitmq_publish.py

```diff
@@ -155,11 +182,15 @@ def main():
         durable=dict(default=False, type='bool'),
         exclusive=dict(default=False, type='bool'),
         auto_delete=dict(default=False, type='bool'),
-        headers=dict(default={}, type='dict')
+        headers=dict(default={}, type='dict'),
+        cafile=dict(type='str', required=False),
+        certfile=dict(type='str', required=False),
+        keyfile=dict(type='str', required=False)
```

Comment:
If you add
```suggestion
        keyfile=dict(type='str', required=False),
```
then the next diff will be simpler. (Here, the `headers` line was touched even though the PR does not concern that line.)

Reply:
Thanks for the pro tip.  I've added this in for next time.